### PR TITLE
bottom navigation tabs are removed for AppLayout

### LINKED ISSUE #6565: AppLayout: Remove bottom tabs from home screen
in new design we won't have any tabs on home screen

### PR DESCRIPTION
**Pull request by fedrunov**
## Type of change

- [x] Feature
- [ ] Bugfix
- [ ] Technical
- [ ] Other :

## Content

removes bottom navigation tabs from new home screen

## Motivation and context

part of #6502
closes #6565

### CODE REVIEW

**Review by ericdecanini — APPROVED** [review]

File: vector/src/main/java/im/vector/app/features/home/NewHomeDetailFragment.kt [hunk 1]
```diff
@@ -124,24 +121,18 @@ class NewHomeDetailFragment @Inject constructor(
...
-        setupBottomNavigationView()
         setupToolbar()
         setupKeysBackupBanner()
         setupActiveCallView()
 
-        withState(viewModel) {
-            // Update the navigation view if needed (for when we restore the tabs)
-            views.bottomNavigationView.selectedItemId = it.currentTab.toMenuId()
+        childFragmentManager.commitTransaction {
+            add(R.id.roomListContainer, HomeRoomListFragment::class.java, null, "HOME_ROOM_LIST")
```
Comment (ganfra) [inline vector/src/main/java/im/vector/app/features/home/NewHomeDetailFragment.kt:129]: Create constant for "HOME_ROOM_LIST"?

**Review by ganfra — APPROVED** [review]
Small remark otherwise LGTM

**Comment by DoM1niC** [comment]
the Appbar issue the Fab Buttons here ...
grafik

**Comment by fedrunov** [comment]
> Appbar / Navbar issue the Fab Buttons here ... grafik

There is a separate issue for this - https://github.com/vector-im/element-android/issues/6765. This PR does not cause this  problem, it just makes fabs be lower than now and so they are cut off instead of bottom navigation bar

**Review by ganfra — APPROVED** [review]
LGTM

**Pull request merged**